src/main/kotlin/com/exactpro/th2/codec/fixng/FixNgCodecSettings.kt

```diff
@@ -23,5 +23,6 @@ data class FixNgCodecSettings(
     val beginString: String = "FIXT.1.1",
     val dictionary: String,
     val charset: Charset = Charsets.US_ASCII,
-    val dirtyMode: Boolean = false
-) : IPipelineCodecSettings
+    val dirtyMode: Boolean = false,
+    val useStringValueFormat: Boolean = true
```

Comment:
suggestion: Add a comma at the end of the line. In this case, when you add a new parameter in the future the line won't be changed